source/assets/javascripts/application.js

```diff
@@ -20,3 +20,9 @@ if (waypointElement !== null) {
 if (/iPad|iPhone|iPod/.test(navigator.userAgent)) {
   document.documentElement.className += " ios-device";
 }
+
+jQuery.get("http://ipinfo.io", function(response) {
+  if (response.country == 'ZA') {
+    $("#London").insertAfter("#CapeTown");
+  }
+}, "jsonp");
```

Comment:
I was going to express exactly the same concern about including jQuery. Based on our IE statistics (4.38% of total sessions made - 68.24% of them at IE11 - of which only 4.95% people were browsing from South Africa) I think we should be consider using [`XMLHttpRequest`](https://developer.mozilla.org/en/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest). Here's also a helpful side-by-side comparison of jquery code to vanilla JS: http://youmightnotneedjquery.com/#request